src/slideshow.c

```diff
@@ -473,6 +473,14 @@ char *feh_printf(char *str, feh_file * file, winwidget winwid)
 		if ((*c == '%') && (*(c+1) != '\0')) {
 			c++;
 			switch (*c) {
+	        case 'a':
+	           	if (opt.paused == 1) {
```

Comment:
Indentation looks off?

Reply:
Fixed in e58cea4dea47cce762192ba03eb80eb75fbe16d1, thanks for noticing!